pkg/apis/core/validation/validation_test.go

```diff
@@ -2909,7 +2909,7 @@ func TestValidateCSIPersistentVolumeSource(t *testing.T) {
 	}
 
 	for i, tc := range testCases {
-		errs := validateCSIPersistentVolumeSource(tc.csi, field.NewPath("field"))
+		errs := validateCSIPersistentVolumeSource(tc.csi, false, field.NewPath("field"))
```

Comment:
Please add some unit tests with `true` to check the new behavior.

Reply:
Done. thanks.